benchmark_runner/common/ocp_resources/migrate_infra.py

```diff
@@ -0,0 +1,35 @@
+
+import os
+
+from benchmark_runner.common.oc.oc import OC
+from benchmark_runner.common.logger.logger_time_stamp import logger_time_stamp, logger
+from benchmark_runner.common.ocp_resources.create_ocp_resource_operations import CreateOCPResourceOperations
+
+
+class MigrateInfra(CreateOCPResourceOperations):
+    """
+    This class is create OCP resources
+    """
+    def __init__(self, oc: OC, path: str, resource_list: list, ibm_blk_disk_name: list):
+        super().__init__(oc)
+        self.__oc = oc
+        self.__path = path
+        self.__resource_list = resource_list
+        self.__ibm_blk_disk_name = ibm_blk_disk_name
+
+    @logger_time_stamp
+    def create_odf(self):
+        """
+        This method create odf
+        :return:
+        """
+        for resource in self.__resource_list:
+            logger.info(f'run {resource}')
+            if resource == '01_cluster-monitoring-configmap-template.yaml':
+                nodes = self.__oc.get_master_nodes().split()
+                if len(nodes) < 1:
```

Comment:
What is the purpose of this test?

Reply:
If we don't have any infra nodes, we don't want to try to bind the infra components to infra nodes.